frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelinesDatabaseSection.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  FormSection,
+  Title,
+  FormGroup,
+  ExpandableSection,
+  Radio,
+  Text,
+} from '@patternfly/react-core';
+import React from 'react';
+import DatabaseConnectionField from './DatabaseConnectionField';
+import { PipelineServerConfigType } from './ConfigurePipelinesServerModal';
+
+type PipelinesDatabaseSectionProps = {
+  setConfig: (config: PipelineServerConfigType) => void;
+  config: PipelineServerConfigType;
+};
+
+export const PipelinesDatabaseSection = ({ setConfig, config }: PipelinesDatabaseSectionProps) => {
+  const [databaseIsExpanded, setDatabaseIsExpanded] = React.useState(false);
+
+  return (
+    <FormSection
+      title={
+        <>
+          <Title headingLevel="h2">Database</Title>
+          <Text component="p" className="form-subtitle-text">
+            To store data science pipelines. By default we include a database that will store this
+            on your cluster.
+          </Text>
+        </>
+      }
+    >
+      <FormGroup hasNoPaddingTop isStack>
+        <ExpandableSection
+          isIndented
+          toggleText={`${databaseIsExpanded ? 'Hide' : 'Show'} advanced database options`}
+          onToggle={() => setDatabaseIsExpanded(!databaseIsExpanded)}
+          isExpanded={databaseIsExpanded}
+        >
+          <FormGroup hasNoPaddingTop isStack>
+            <Radio
+              className="checkbox-radio-fix-body-width"
+              name="default-database-type-radio"
+              id="default-database-connection-type-radio"
+              label="Use default database stored on your cluster"
+              isChecked={config.database.useDefault}
+              onChange={() =>
+                setConfig({
+                  ...config,
+                  database: {
+                    ...config.database,
+                    useDefault: true,
+                  },
+                })
+              }
+            />
+            <Radio
+              className="checkbox-radio-fix-body-width"
+              name="external-database-type-radio"
```

Comment:
semantically, your radio buttons should share the same `name`.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#defining_a_radio_group

React allows us to sorta mangle this -- but that isn't a reason to mangle it.

Reply:
ok so both radios should be "database-type-radio"